frontend/institution/base_institution_page_mobile.html

```diff
@@ -0,0 +1,51 @@
+<md-content layout-align="center" hide-navbar="both">
+    <div flex layout="column" md-colors="{background: 'grey-50'}">
+        <!-- COVER -->
+        <div id="cover-inst-mobile">
+           <institution-header title="{{institutionCtrl.getLimitedName(110)}}"
+                photo="institutionCtrl.getPhoto()">
+           </institution-header>
+            <div id="inst-name-mobile">
+                <hr id="divider-institution"/>
+                <div id="buttons-institution-page">
+                    <md-button md-colors="{background: 'teal-500'}"
```

Comment:
create an object on controller with an onClick, icon and name properties. than you can use ng-repeat and get less lines of code.